assemble_app/app/controllers/projects_controller.rb

```diff
@@ -13,6 +13,17 @@ def create
         end
     end
 
+    def update
```

Comment:
Is there validation being done so that someone who is not an owner can update the project?

Reply:
We just have it so the owner has edit access